Resolve getLabels immediately, if free listings are requested. …

tomalec opened the pull request:
### Changes proposed in this Pull Request:
Resolve `getLabels` immediately, if free listings are requested.
Improve the UX, so a user would not have to wait for the paid campaigns to be loaded if they request a free program from a static, hard-coded list.

Improves #651


### Screenshots:

![Screencast that shows free listings label immediately](https://user-images.githubusercontent.com/17435/120664308-1fec2500-c48b-11eb-8b3b-5fcab8f6b872.gif)



### Detailed test instructions:

1. Go to Programs Report page `/wp-admin/admin.php?page=wc-admin&path=%2Fgoogle%2Freports%2Fprograms`
2. Select "Free Listings" by the dropdown filter
3. Reload the page
4. The selected program label should show up in the filter on page load immediately.

tomalec committed:
Resolve `getLabels` immediately, if free listings are requested.

Improve the UX, so a user would not have to wait for the paid campaigns to be loaded if they request a free program from a static, hard-coded list.

eason9487 reviewed (approved):
LGTM. Left a question for double-check and no need to block this PR.

tomalec committed:
Fix condition for resolving free listings label,

in `createProgramsFilterConfig`. Check if selected ids are at least a subset of the free listings programs.
